bugalink-backend/users/tests.py

```diff
@@ -1,79 +1,85 @@
-from locations.models import Location
-from django.test import TestCase
-from rest_framework.test import APIClient
 import json
-from datetime import time, datetime, timedelta
-from .models import User
-from passengers.models import Passenger
-from drivers.models import Driver
+from datetime import datetime, time, timedelta
+
+from django.test import TestCase
 from driver_routines.models import DriverRoutine
-from trips.models import Trip, TripRequest
+from drivers.models import Driver
+from locations.models import Location
+from passengers.models import Passenger
 from ratings.models import DriverRating
+from rest_framework.test import APIClient
+from trips.models import Trip, TripRequest
+
+from .models import User
+
 
 def load_data(self):
-    self.user = User.objects.create(email="test@test.com", first_name="nameTest", last_name="lastNameTest", is_passenger = True, is_driver=True)
+    self.user = User.objects.create(
+        email="test@test.com",
+        first_name="nameTest",
+        last_name="lastNameTest",
+        is_passenger=True,
+        is_driver=True,
+    )
     self.passenger = Passenger.objects.create(user=self.user)
     self.driver = Driver.objects.create(user=self.user)
 
+
 def load_complex_data(self):
-    '''
+    """
     Crea:
     * self.driver_routine para self.driver
-    * self.trip(FINISHED) y self.trip_2(PENDING) para self.driver_routine 
+    * self.trip(FINISHED) y self.trip_2(PENDING) para self.driver_routine
     * self.user_2 otro usuario con passenger_2 y driver_2
     * self.trip_request(ACCEPTED) para passenger_2 en self.trip
     * self.trip_request_2(PENDING) para passenger_2 en self.trip_2
-    '''
+    """
     load_data(self)
-    self.user_2 = User.objects.create(email="test2@test.com", first_name="Second User", last_name="lastNameTest2", is_passenger = True, is_driver=True)
+    self.user_2 = User.objects.create(
+        email="test2@test.com",
```

Comment:
Please, install pre-commit hooks and it will code and lint your code automatically, this does not follow our formatting conventions

Reply:
i do have it installed already, it should have been reformatted